standard_rules/unusual_login.py

```diff
@@ -111,3 +119,14 @@ def title(event):
         f" from {GEO_INFO.get('city')}, {GEO_INFO.get('region')} in {GEO_INFO.get('country')}"
         f" (not in last [{GEO_HISTORY_LENGTH}] login locations)"
     )
+
+
+def alert_context(event):
+    context = {}
+    context["ip"] = event.udm("source_ip")
+    context["reverse_lookup"] = GEO_INFO.get("hostname", "No reverse lookup hostname")
+    context["ip_org"] = GEO_INFO.get("org", "No organization listed")
+    if GEO_HISTORY:
```

Comment:
For future proofing (in case things get parallelized in the backend rather than running the title/context/aux function right after the rule), we could consider making the geo history a dictionary that keys off `p_row_id` Similar to [this](https://github.com/panther-labs/panther-analysis/blob/master/okta_rules/okta_geo_improbable_access.py#L121)

```
GEO_HISTORY[event.get('p_row_id')] = updated_geo_logins

...

if GEO_HISTORY[event.get('p_row_id')]:
     context["geoHistory"] = f"{json.dumps(GEO_HISTORY[event.get('p_row_id')])}"
```

Reply:
Done!